packages/react/src/components/Table/TableSkeletonWithHeaders/TableSkeletonWithHeaders.jsx

```diff
@@ -42,42 +46,57 @@ const TableSkeletonWithHeaders = ({
   hasRowActions,
   columns,
   rowCount,
+  columnCount,
   testID,
   testId,
   showExpanderColumn,
-}) => (
-  <TableBody data-testid={testID || testId}>
-    <TableRow className={`${iotPrefix}--table-skeleton-with-headers--table-row`}>
-      {hasRowSelection === 'multi' ? <TableCell /> : null}
-      {hasRowExpansion ? <TableCell /> : null}
-      {columns.map((column) => (
-        <TableCell key={`skeletonCol-${column.id}`}>
-          <SkeletonText />
-        </TableCell>
-      ))}
-      {showExpanderColumn ? (
-        <TableCell data-testid={`${testID || testId}-expander-column`} />
-      ) : null}
-      {hasRowActions ? <TableCell /> : null}
-    </TableRow>
-    {[...Array(rowCount > 0 ? rowCount - 1 : 0)].map((row, index) => (
+}) => {
```

Comment:
Note this indentation change will be much easier to review if ignore whitespace changes are turned on.

Reply:
Thanks, I wouldn't mind having that on by default if possible